src/nodes.coffee

```diff
@@ -687,12 +686,20 @@ exports.Call = class Call extends Base
 
     fragments = []
     if this instanceof SuperCall
-      preface = @superReference(o) + ".call(#{@superThis(o)}"
-      if compiledArgs.length then preface += ", "
+      preface = @superReference o
+      if preface isnt 'super'
```

Comment:
Bare `super` (e.g. in a constructor) isn't a function reference, and needs special treatment.